packages/slice-machine/tests/components/CreateCustomTypeModal.spec.tsx

```diff
@@ -14,6 +14,7 @@ import { AnalyticsBrowser } from "@segment/analytics-next";
 import Tracker from "../../src/tracker";
 
 jest.mock("next/dist/client/router", () => require("next-router-mock"));
+global.console = { ...global.console, error: jest.fn() };
```

Comment:
yeah you might not want to over-ride the global console.error, as that's how bad tests are reported using a spy should work :)